### PR TITLE
To k8s client 600

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: testing/core-junit/src/main/java/io/dekorate/testing/CustomResourceCondition.java [hunk 1]
```diff
@@ -28,10 +29,14 @@
...
+import io.fabric8.kubernetes.client.http.HttpRequest;
+import io.fabric8.kubernetes.client.http.HttpResponse;
 import io.fabric8.kubernetes.client.utils.URLUtils;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
+
+//import okhttp3.OkHttpClient;
+//import okhttp3.Request;
+//import okhttp3.Response;
```
Comment (iocanel) [inline testing/core-junit/src/main/java/io/dekorate/testing/CustomResourceCondition.java:39]:
```suggestion
```

File: testing/core-junit/src/main/java/io/dekorate/testing/CustomResourceCondition.java [hunk 2]
```diff
@@ -53,7 +58,9 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext con
 
       KubernetesClient client = getKubernetesClient(context);
       Config config = client.getConfiguration();
-      OkHttpClient http = client.adapt(OkHttpClient.class);
+      //      OkHttpClient http = client.adapt(OkHttpClient.class);
```
Comment (iocanel) [inline testing/core-junit/src/main/java/io/dekorate/testing/CustomResourceCondition.java:61]:
```suggestion
```

File: testing/core-junit/src/main/java/io/dekorate/testing/CustomResourceCondition.java [hunk 3]
```diff
@@ -71,9 +78,13 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext con
       }
       parts.add(plural);
       String requestUrl = URLUtils.join(parts.stream().toArray(s -> new String[s]));
-      Request request = new Request.Builder().get().url(requestUrl).build();
-      Response response = http.newCall(request).execute();
+      //      Request request = new Request.Builder().get().url(requestUrl).build();
+      //      Response response = http.newCall(request).execute();
```
Comment (iocanel) [inline testing/core-junit/src/main/java/io/dekorate/testing/CustomResourceCondition.java:82]:
```suggestion
      //      Request request = new Request.Builder().get().url(requestUrl).build();
      //      Response response = http.newCall(request).execute();
```

**Review by iocanel — APPROVED** [review]
LGTM.

Please commit the suggestion that completely remove commented out code and it is good to go.

**Comment by iocanel** [comment]
@aureamunoz before mreging we also need to squash the  commits

**Comment by aureamunoz** [comment]
This finally passed @iocanel !

**Pull request merged**